.travis.yml

```diff
@@ -118,25 +118,10 @@ jobs:
 script:
   - export RUN_JOB=1
```

Comment:
Why isn't setting RUN_JOB in travis shell? The code inside there looks for this value and only there.

Reply:
Make sense, as now all logic is in Travis shell